client/jetpack-cloud/sections/agency-dashboard/sidebar/index.tsx

```diff
@@ -13,6 +13,7 @@ import { useDispatch } from 'calypso/state';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import 'calypso/components/jetpack/sidebar/style.scss';
 import type { FunctionComponent } from 'react';
+import './style.scss';
```

Comment:
I see we have imported the styles already: import 'calypso/components/jetpack/sidebar/style.scss';

Can we keep one?

Also,

We usually keep a one-line space before the style import: `import './style.scss';`

Example"

```
import type { FunctionComponent } from 'react';

import './style.scss';
```

Reply:
I re-ordered the imports, but I kept the new rules inside `./style.scss`. My reasoning: these new rules are relevant _only_ to the Agency Dashboard, so including them elsewhere would introduce extra unused styling code.

Reply:
Sounds good 👍 I actually thought we were importing the same file initially.